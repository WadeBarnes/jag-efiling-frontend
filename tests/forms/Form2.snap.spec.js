
import ShallowRenderer from 'react-test-renderer/shallow';
require('../support/enzyme.setup');
import React from 'react';
import Form2 from '../../src/forms/Form2';

describe('Form 2 snapshot', ()=>{

    test('snapshot', () => {

        const renderer = new ShallowRenderer();
        const tree = renderer.render(<Form2 homePath={"/"}/>);
        expect(tree).toMatchSnapshot();
    })

});
