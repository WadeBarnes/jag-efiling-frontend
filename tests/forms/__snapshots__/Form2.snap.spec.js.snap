// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form 2 snapshot snapshot 1`] = `
<div
  className="template container gov-container form"
  id="topicTemplate"
>
  <div
    id="breadcrumbContainer"
  >
    <ol
      className="breadcrumb"
    >
      <li>
        <a
          href="/"
          id="home"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href=""
        >
          Notice of Appearance (Form 2)
        </a>
      </li>
    </ol>
  </div>
  <ProgressStatusBar
    activeStep={1}
    steps={
      Array [
        "Access",
        "Form 2",
        "Preview",
        "Payment",
      ]
    }
  />
  <div
    className="row"
  >
    <div
      className="contentPageMainColumn col-sm-12"
      id="main-content"
      role="main"
    >
      <div
        className="form-title not-printable"
      >
        <h1>
          NOTICE OF APPEARANCE
        </h1>
        Form 2 (RULES 5 (A), 13 (A) AND 17 (A))
      </div>
      <div
        className="form-section not-printable"
      >
        <h2
          style={
            Object {
              "fontWeight": "bold",
            }
          }
        >
          When Do You Use the Notice of Appearance (Form 2)?
        </h2>
        <p>
          If you have been served a Notice of Appeal (Form 7), and you want to respond, fill out the Notice of Appearance (Form 2). If you do not respond, you will not be included in future court proceedings related to this case, and will not have access to the case documents filed.
        </p>
        <p>
          You have 
          <span
            style={
              Object {
                "fontWeight": "bold",
              }
            }
          >
            10 days
          </span>
           to serve a filed copy of the Notice of Appearance to the appellant, after receiving a Notice of Appeal.
        </p>
      </div>
      <Find
        callback={[Function]}
        formSevenNumber="CA"
        handleFieldChange={[Function]}
        notFoundError=""
        startSearching={[Function]}
      />
      <div
        className="form-section"
        id="form-data-section"
        style={
          Object {
            "display": "none",
          }
        }
      >
        <Form2DataSection
          closeForm={[Function]}
          data={
            Object {
              "appellants": Array [
                Object {
                  "address": Object {
                    "addressLine1": "",
                    "addressLine2": "",
                    "city": "",
                    "postalCode": "",
                    "province": "",
                  },
                  "name": "",
                },
              ],
              "email": "",
              "phone": "",
              "respondents": Array [
                Object {
                  "address": Object {
                    "addressLine1": "",
                    "addressLine2": "",
                    "city": "",
                    "postalCode": "",
                    "province": "",
                  },
                  "name": "",
                  "selected": false,
                },
              ],
              "selectedContactIndex": 0,
              "sendNotifications": false,
              "useServiceEmail": false,
            }
          }
          handleFieldChange={[Function]}
          saveForm={[Function]}
          show={false}
          validate={[Function]}
        />
        <FormButtonBar
          back={[Function]}
          disablePreview={true}
          formErrorMessage=""
          preview={[Function]}
          save={[Function]}
        />
      </div>
      <div
        className="modal not-printable"
        id="dataLossWarning"
        style={
          Object {
            "display": "none",
          }
        }
      >
        <div
          className="modal-title warning"
        >
          <span
            id="close-modal"
            onClick={[Function]}
          >
            Ã—
          </span>
          Warning!
        </div>
        <div
          className="modal-content"
        >
          <div>
            All changes will be lost!
          </div>
          <FormButtonBar
            back={[Function]}
            backMessage="Go back so I can save the form as draft."
            continue={[Function]}
            continueMessage="Continue anyway.  I don't care about the data."
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
